deployment&configuration/netsuite/flows/shipping-method-mapping/readme.md

```diff
@@ -0,0 +1,38 @@
+# Shipping Method Mapping between HotWax Commerce and NetSuite
+
+The shipping method mapping between HotWax Commerce and NetSuite ensures that the shipping methods selected in Shopify correspond accurately to their counterparts in both HotWax Commerce and NetSuite. HotWax Commerce facilitates the mapping of shipping methods chosen from Shopify, providing accurate information to NetSuite. This process establishes coherence across platforms, maintaining consistency in order processing and fulfillment workflows. 
+
+## Creating a Shipping Method in HotWax Commerce:
+
+Shipping methods that are supported by the retailer needs to be created in HotWax Commerce to ensure that the shipping information is sent to the Shipping carrier. Shipping methods can be created by following these steps:
```

Comment:
Should have some external system context, as the order will go to Netsuite and the Shipping method needs to be presented there